File: pset6/cash/cash.py
coins = (25, 10, 5, 1)
while 1:
    try:
        owed = float(input('Change owed: '))
        if owed >= 0 and round(owed, 2) == round(owed, 3):
            break
    except ValueError:
        pass
owed *= 100
num_changes = 0
while owed > 0:
    owed -= 25 if owed >= 25 else 10 if owed >= 10 \
        else 5 if owed >= 5 else 1
    num_changes += 1
print(num_changes)

File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <ctype.h>

int main(void)
{
    float owed = -1.0;
    while (owed < 0)
    {
        owed = get_float("Change owed: ");
    }
    int coins = 0, left = round(owed * 100);
    while (left)
    {
        int change = left >= 25 ? 25 : left >= 10 ? 10 : left >= 5 ? 5 : 1;
        left -= change;
        coins++;
    }
    printf("%d\n", coins);
}
